common/collections.ts

```diff
@@ -146,6 +150,10 @@ export class Bytes extends ByteArray {
   static fromUint8Array(uint8Array: Uint8Array): Bytes {
     return changetype<Bytes>(uint8Array)
   }
+
+  static zero(): Bytes {
```

Comment:
I assume this has length 0, correct? In which case it makes more sense to call this `empty` instead of `zero`.

Reply:
Good point!